python/lsst/ctrl/bps/wms/panda/conf_example/pipelines_check_idf.yaml

```diff
@@ -2,15 +2,12 @@ pipelineYaml: "${OBS_SUBARU_DIR}/pipelines/DRP.yaml#processCcd"
 
 payload:
   payloadName: pipelines_check
-  runInit: true
-  output: "u/{operator}/{payload_name}"
-  outCollection: "{output}/{timestamp}"
   butlerConfig: s3://butler-us-central1-panda-dev/hsc/butler.yaml
   inCollection: HSC/calib,HSC/raw/all,refcats
   dataQuery: "tract = 9615 and patch=30 and detector IN (10..11) and instrument='HSC' and skymap='hsc_rings_v1' and band in ('r')"
+  output: u/{operator}/pcheck_ebutler
 
-  sw_image: "spodolsky/centos:7-stack-lsst_distrib-d_2021_09_06"
+  sw_image: "lsstsqre/centos:7-stack-lsst_distrib-w_2021_39"
   fileDistributionEndPoint: "s3://butler-us-central1-panda-dev/hsc/{payload_folder}/{uniqProcName}/"
   s3_endpoint_url: "https://storage.googleapis.com"
```

Comment:
Do sw_image, fileDistributionEndPoint, and s3_endpoint_url change often?  If not, they could go in the central config as defaults.

Reply:
Can the sw_image be defaulted by the plugin to one matching the submit version (still allow user to override it but only if they need to)?

Reply:
I think that's a very good idea.